jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/spat/SpatTopology.java

```diff
@@ -46,10 +46,14 @@ public static Topology build(String spatOdeJsonTopic, String spatProcessedJsonTo
             rawOdeSpatStream.mapValues(
                 (Void key, Bytes value) -> {
                     DeserializedRawSpat deserializedRawSpat = new DeserializedRawSpat();
-                    JsonValidatorResult validationResults = spatJsonValidator.validate(value.get());
-                    deserializedRawSpat.setOdeSpatOdeSpatData(JsonSerdes.OdeSpat().deserializer().deserialize(spatOdeJsonTopic, value.get()));
-                    deserializedRawSpat.setValidatorResults(validationResults);
-                    logger.debug(validationResults.describeResults());
+                    try {
+                        JsonValidatorResult validationResults = spatJsonValidator.validate(value.get());
+                        deserializedRawSpat.setOdeSpatOdeSpatData(JsonSerdes.OdeSpat().deserializer().deserialize(spatOdeJsonTopic, value.get()));
+                        deserializedRawSpat.setValidatorResults(validationResults);
+                        logger.debug(validationResults.describeResults());
+                    } catch (Exception e) {
+                        logger.error("Error in mapValues:", e);
```

Comment:
Same comment as for the Map topology: Attach error message or return null, not empty DeserializedRawSpat.